scripts/build/osDistro.sh

```diff
@@ -27,7 +27,8 @@ centos_distro=$(cat /etc/redhat-release 2> /dev/null | awk '{print $3}')
 # Instead the one in previous versions:
 # CentOS release 6.6 (Final)
 # The check in the next line ensure that the distro version is correctly detected in any case.
-if [ $centos_distro == "release" ]; then
+if [ "$centos_distro" = "release" ]
```

Comment:
It seems that `==` don't work with `sh` (it works with `bash`). See http://stackoverflow.com/questions/10849297/compare-a-string-in-unix#comment14133719_10849357.

NTC